kin-sdk-core/src/androidTest/java/kin/sdk/core/BaseTest.java

```diff
@@ -3,26 +3,54 @@
 import android.content.Context;
 import android.support.test.InstrumentationRegistry;
 
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.io.InputStream;
 import org.junit.After;
 import org.junit.Before;
 
 import java.io.File;
 
 public class BaseTest {
 
-    private final String PARITY_PROVIDER_URL = "http://207.154.247.11:8545";
+    /**
+     * The Computer localhost on Android Studio emulator is 10.0.2.2
+     * The Computer localhost on Genymotion emulator is 10.0.3.2
+     */
+    private final String TESTRPC_PROVIDER_URL = "http://10.0.2.2:8545";
 
     private Context context;
     private ServiceProvider serviceProvider;
-    protected KinClient kinClient;
+    KinClient kinClient;
+    Config config;
 
     @Before
     public void setUp() throws Exception {
         context = InstrumentationRegistry.getContext();
-        serviceProvider = new ServiceProvider(PARITY_PROVIDER_URL, ServiceProvider.NETWORK_ID_ROPSTEN);
+        serviceProvider = new ServiceProvider(TESTRPC_PROVIDER_URL, ServiceProvider.NETWORK_ID_TRUFFLE);
+        getConfigFile();
+        clearKeyStore();
         kinClient = new KinClient(context, serviceProvider);
     }
 
+
+    private void getConfigFile() {
+        String json = null;
+        try {
+            InputStream is = context.getAssets().open("testConfig.json");
+            int size = is.available();
+            byte[] buffer = new byte[size];
+            is.read(buffer);
+            is.close();
+            json = new String(buffer, "UTF-8");
+        } catch (IOException ex) {
+            ex.printStackTrace();
+        }
+        config = new Gson().fromJson(json, Config.class);
```

Comment:
Alternatively I think you can do this 
```java
InputStreamReader inputStreamReader = new InputStreamReader(context.getAssets().open("example.txt"));
JsonReader reader = new JsonReader(new FileReader(inputStreamReader));
config = new Gson().fromJson(reader, Config.class);
```

Reply:
why do we need an alternative?

Reply:
Good question 😃  👍  
For a small configuration file it is fine as is. 
My suggested alternative would have been better for a larger file, with the addition of wrapping FileReader with a BufferedReader

Reply:
shouldn't be a large file.